directory_header_footer/context_processors.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+
+
+def sso_processor(request):
+    url = request.build_absolute_uri()
+    sso_password_reset_url = getattr(settings, 'SSO_PASSWORD_RESET_URL', None)
```

Comment:
`SSO_PASSWORD_RESET_URL` is only used by sso profile.

I will move `SSO_PASSWORD_RESET_URL` to sso profile context processor.